homework/hackyourtemperature/__test__/app.test.js

```diff
@@ -0,0 +1,28 @@
+import supertest from "supertest";
+import app from "../app.js";
+
+const request = supertest(app);
+
+describe("POST /weather", () => {
+  it("404 error if the city Name is not found", async () => {
```

Comment:
Maybe we can add 'responds with' to the beginning of each test to align tests with behaviour driven development.

```suggestion
  it("responds with 404 error if the city Name is not found", async () => {
```

Ref: https://en.wikipedia.org/wiki/Behavior-driven_development

Reply:
Done!